spec/feature/registeration_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'capybara/rspec'
+
+feature "Chitter user behaviour" do
+
+  scenario "Users can sign up for Chitter" do
+    visit '/'
+    click_link "Sign Up"
+
+    expect { sign_up }.to change(User, :count).by(1)
+    expect(User.count).to eq(1)
+
+    user = User.first
+    expect(user.name).to eq("John Bosco")
+    expect(user.username).to eq("Chitterer")
+  end
+
+  scenario "but must use a unique username and email" do
```

Comment:
Your scenario is worded `"but must use a unique username and email"`.  the `but` is in reference to the previous scenario `"Users can sign up for Chitter"`.  You shouldn't do this.